src/rezplugins/shell/_utils/powershell_base.py

```diff
@@ -280,7 +280,9 @@ def appendenv(self, key, value):
             .format(key, os.path.pathsep, value))
 
     def unsetenv(self, key):
-        self._addline(r"Remove-Item Env:\%s" % key)
+        self._addline(
+            'Remove-Item -ErrorAction -SilentlyContinue "Env:{0}"'.format(key)
```

Comment:
I think there's an extra dash here that shouldn't be there, I got this:
```
Line |
 913 |  Remove-Item -ErrorAction -SilentlyContinue "Env:CUDA_CACHE_MAXSIZE"
     |              ~~~~~~~~~~~~
     | Cannot bind parameter 'ErrorAction'. Cannot convert value "-SilentlyContinue" to type
     | "System.Management.Automation.ActionPreference". Error: "Unable to match the identifier name -SilentlyContinue
     | to a valid enumerator name. Specify one of the following enumerator names and try again: SilentlyContinue, Stop,
     | Continue, Inquire, Ignore, Suspend, Break"

Reply:
Yeah it should just be `SilentlyContinue` without the hyphen